### ISSUE TITLE
Android 11 Issue

### ISSUE THREAD

**Issue by rizwankhanMiranshah** [nbsp-team/MaterialFilePicker#128]
Input dispatching timed out (Waiting to send key event because the focused window has not finished processing all of the input events that were previously delivered to it. Outbound queue length: 0. Wait queue length: 1.)

**Comment by pradeep-orbi** [comment]
Is this library working on android 11 devices ?

**Comment by ArpitFuturism** [comment]
Yes this library supports android 11 too. Just you need to ask the MANAGE_EXTERNAL_STORAGE permission for android 11
